.github/workflows/pipeline-segment-electron-build.yml

```diff
@@ -114,7 +114,7 @@ jobs:
     - name: Get Depot Tools
       timeout-minutes: 5
       run: |
-        git clone --depth=1 https://chromium.googlesource.com/chromium/tools/depot_tools.git
+        git clone --filter=tree:0 https://chromium.googlesource.com/chromium/tools/depot_tools.git
```

Comment:
This causes an error applying the gclient patch:
https://github.com/electron/electron/actions/runs/10048493604/job/27782411242?pr=42779#step:9:29

Reply:
Hey @jkleinsc just saw your comment after the PR is merged. Should we revert this PR?

Reply:
or wait until https://github.com/electron/electron/pull/42977 merges with the correct configs